### PR TITLE
Misc fixes for Cairn translator

### PR DESCRIPTION
**Pull request by symac**
Have been fixed : 
- detectWeb was missing book and bookSection and sometimes others;
- PDF download was not working following a change on cairn's website
- some metadata have been added because not correctly encoded (page number, publication date)

### CODE REVIEW

**Comment by zuphilip** [comment]
Related discussion on Twitter: https://twitter.com/amarois/status/928653241365262338

**Comment by zuphilip** [comment]
Thank you for this pull request, @symac. 🚀 

I just did some cleanup and small improvements. Please have a look.

Do you have an example for a book chapter?

(Maybe we should also consider to take the RIS data that Cairn provides under `Citer cet article` at some point...)

**Comment by symac** [comment]
Thanks @zuphilip for the cleanup. A book chapter example is http://www.cairn.info/jeu-d-echecs-comme-representation--9782728835904-page-111.htm

I think this .ris is quite new, I have never noticed it. That would maybe be a better way to get data yes, but I have not much time at the moment to check if it is complete so I believe that might wait :)

File: Cairn.info.js [hunk 1]
```diff
@@ -31,17 +31,25 @@
...
+	} else if (breadcrumbPage == "Article") {
 		return "journalArticle";
+	} else if (breadcrumbPage == "Chapitre") {
+		return "bookSection";
+	} else if (breadcrumbPage == "Résumé") {
+		typeDocument = ZU.xpathText(doc, '//div[@id="breadcrump"]/a[2]');
+		if (typeDocument == "Revues") {
+			return "journalArticle";
+		} else if (typeDocument == "Ouvrages") {
+			return "book";
```
Comment (zuphilip) [inline Cairn.info.js]: I guess this should be "bookSection" e.g. your example http://www.cairn.info/jeu-d-echecs-comme-representation--9782728835904-page-111.htm

**Pull request merged**